ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -323,14 +329,15 @@ def _generate_result(result_file, *, failure_message=None, skip=False, error_mes
         '<skipped type="skip" message="">![CDATA[Test Skipped by developer]]</skipped>' \
         if skip else ''
     return """<?xml version="1.0" encoding="UTF-8"?>
-<testsuite name="%s" tests="1" failures="%d" time="0" errors="%d" skipped="%d">
+<testsuite name="%s" tests="1" failures="%d" time="%f" errors="%d" skipped="%d">
   <testcase classname="%s" name="%s.missing_result" time="0">
```

Comment:
Shouldn't the time of the `testcase` be set to the same value?

Reply:
See 208ef70